src/main/java/com/github/lunatrius/laserlevel/marker/Marker.java

```diff
@@ -10,6 +10,7 @@
     public boolean enabled;
     public int spacing;
     public int rgb;
+    public int maxdistance;
```

Comment:
Nitpicking, but `maxDistance` would be a better name for this field (json/lang literals are fine as they are just data).